### ISSUE TITLE
Sentinel Nodes stop working

### ISSUE THREAD

**Issue by Femkecl** [sentinel-official/dvpn-node#87]
**Describe the bug**
Nodes stop working for some reason randomly.
With no explanation at all. I'm not the only one with this issue nodes stop working because of the error in the picture below

**To Reproduce**
Just run the node.

**Expected behavior**
A running node with 100% uptime

**Screenshots**
![error](https://user-images.githubusercontent.com/95861442/148069848-495278bb-4f98-4519-8ec6-c39041088fb0.png)
![333](https://user-images.githubusercontent.com/95861442/148069862-ed171ef6-2921-4736-ad9e-30a274c1cb65.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 LTS

**Additional context**
It seems to have started a bit before the beginning of the New Year.

**Comment by bsrinivas8687** [comment]
Hi @Femkecl It seems like the issue is with RPC endpoint. Can you please try checking with another public RPC endpoint.

**Comment by Femkecl** [comment]
Have done that and the issue remains.

**Comment by HammontreeArt** [comment]
Please, question regarding your comment "try checking with another public RPC endpoint"

According to the documentation we are advised to use "https://rpc.sentinel.co:443".
![image](https://user-images.githubusercontent.com/90418397/150375853-f3780a56-5ccc-4f50-9004-6c2dc28e43a3.png)

Please, are you able to provide an authorized list of alternate RPC endpoints?

B.Rgds,
HA

**Comment by qf3l3k** [comment]
One thing will be to have alternate RPC servers. However, at the moment we can provide single RPC endpoint in node configuration. As this seems to be quite critical point in whole setup, maybe reasonable is to consider ability to add up to 3 RPC endpoints and add some round-robin "when first one responds with error, try another one".
That would make whole process more resilient. 
Another solution could be ignoring error for some time and retry RPC communication after let's say 3 or 5 minutes (this interval might also be defined in config "In case of failure retry in X minutes" parameter.
Basically just implement something which prevents immediate node crash once RPC can't be contacted.

**Comment by bsrinivas8687** [comment]
Yes, this will be added in next version.

**Comment by HammontreeArt** [comment]
FYI.  I switched my nodes to a different RPC and so far it's working great.

rpc_address = "https://rpc.sentinel.smartnodes.one:443"

**Comment by vatsansp** [comment]
Hi there, what URL is this? The listing of the nodes.

> **Screenshots** ![error](https://user-images.githubusercontent.com/95861442/148069848-495278bb-4f98-4519-8ec6-c39041088fb0.png) ![333](https://user-images.githubusercontent.com/95861442/148069862-ed171ef6-2921-4736-ad9e-30a274c1cb65.png)

**Comment by prshaka** [comment]
> FYI. I switched my nodes to a different RPC and so far it's working great.
> 
> rpc_address = "https://rpc.sentinel.smartnodes.one:443"

This rpc address seems to be down. I get a timeout at startup.

**Comment by prshaka** [comment]
> Yes, that one died last month. Switch back to the original or try the ITA Staker.
> 
> ![image](https://user-images.githubusercontent.com/90418397/161137865-1babee31-182c-4dbe-bc90-350384d8b5e3.png)

I just found another one but dit not try it rpc.mathnodes.com:4444
I'm using the first one.
